docs/source/index.rst

```diff
@@ -50,6 +50,10 @@ PROJ
         removed from PROJ in version 7.0.0 scheduled for release February 1st
         2020.
 
+    .. attention::
+
+        With the introduction of PROJ 5 behavioural changes has been made to
```

Comment:
I've noticed mostly British spelling throughout other PROJ docs.

Reply:
Probably because I have authored a big chunk of the docs.

@rouault can your `fix_typo` script be tuned to a specific spelling of the English language?